contracts/contracts/strategies/NativeStaking/FeeAccumulator.sol

```diff
@@ -0,0 +1,39 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import { Address } from "@openzeppelin/contracts/utils/Address.sol";
+
+import { Governable } from "../../governance/Governable.sol";
+
+/**
+ * @title Fee Accumulator for Native Staking SSV Strategy
+ * @notice Receives execution rewards which includes tx fees and
+ * MEV rewards like tx priority and tx ordering.
+ * It does NOT include swept ETH from beacon chain consensus rewards or full validator withdrawals.
+ * @author Origin Protocol Inc
+ */
+contract FeeAccumulator is Governable {
```

Comment:
It seems this contract missing a receive `payable` function?

Reply:
Thanks this is a great find. Fixed in this commit: https://github.com/OriginProtocol/origin-dollar/pull/2053/commits/3fb038a61f7275665e689df8e812e590c7b56394